examples/Duration/Main.elm

```diff
@@ -202,12 +202,12 @@ adjustAllowedTimesOfDayToClientZone baseZone clientZone today datetimeBeingProce
 
         startPartsAdjustedForBaseZone =
             Time.posixToParts baseZone datetimeBeingProcessed
-                |> (\parts -> Time.partsToPosix baseZone { parts | hour = 8, minute = 0 })
+                |> (\parts -> Time.partsToPosix baseZone { parts | hour = 22, minute = 0 })
                 |> Time.posixToParts clientZone
 
         endPartsAdjustedForBaseZone =
             Time.posixToParts baseZone datetimeBeingProcessed
-                |> (\parts -> Time.partsToPosix baseZone { parts | hour = 17, minute = 30 })
+                |> (\parts -> Time.partsToPosix baseZone { parts | hour = 21, minute = 59 })
                 |> Time.posixToParts clientZone
```

Comment:
What is the reasoning behind a day being from 22:00 to 21:59 (the next day)? Why not 0:00 to 23:59?